### DISCUSSION TITLE
Single-target workflow?

### DISCUSSION THREAD

**Discussion by sowelipililimute** [WeblateOrg/weblate#7249] [Q&A]
Is it possible to have weblate work on pofiles in a single-target manner?

I have a single repository that's just `pofiles/projectA.po`, `pofiles/projectB.po`, and so on.
Each of these projectA and projectB and so on are files containing source strings that are continually updated by external tooling, and the target strings provided by translators.

Ideally, I'd like to be able to create components for `pofiles/projectA.po`, `pofiles/projectB.po`, that only operate on the source + the target language, instead of the "default" workflow where you provide source .pot templates for Weblate to generate target language versions of.

**Comment by nijel** [comment]
You don't need POT files to translate PO files. Just disable creating new translations withing component configuration and not POT file is needed.
